preparedata/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/preparedata/bdtopo/BDTopoGISLayers.groovy

```diff
@@ -112,6 +112,32 @@ IProcess importPreprocess(){
             def tmp_imperv_surface_activite = 'TMP_IMPERV_SURFACE_ACTIVITE_' + uuid
             def input_impervious = 'INPUT_IMPERVIOUS'
 
+            // -------------------------------------------------------------------------------
+            // Control the SRIDs from input tables
+
+            def list = [tableBuildIndifName, tableIrisName, tableBuildIndusName, tableBuildRemarqName,
+                        tableRoadName, tableRailName, tableHydroName, tableVegetName,
+                        tableImperviousSportName, tableImperviousBuildSurfName,
+                        tableImperviousRoadSurfName, tableImperviousActivSurfName]
+
+            // The SRID is stored and initialized to -1
+            def srid = -1
```

Comment:
default SRID is 0

Reply:
We chose -1 to show that precisely this is a value that will be modified